app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/DraftReturn.scala

```diff
@@ -41,7 +41,7 @@ final case class SingleDisposalDraftReturn(
   reliefDetailsAnswers: Option[ReliefDetailsAnswers],
   exemptionAndLossesAnswers: Option[ExemptionAndLossesAnswers],
   yearToDateLiabilityAnswers: Option[YearToDateLiabilityAnswers],
-  initialGainOrLoss: Option[AmountInPence],
+  initialGainOrLoss: Option[AmountInPenceWithSource],
```

Comment:
and this should also just be `AmountInPence`

Reply:
Done